packages/foam-vscode/src/features/create-from-template.ts

```diff
@@ -51,9 +60,21 @@ For a full list of features see [the VS Code snippets page](https://code.visuals
 2. create a note from this template by running the \`Foam: Create New Note From Template\` command
 `;
 
-async function getTemplates(): Promise<string[]> {
+async function templateMetadata(templateUri: Uri): Promise<object> {
+  const contents = await workspace.fs
+    .readFile(templateUri)
+    .then(bytes => bytes.toString());
+  let templateMetadata: object, _templateWithFoamFrontmatterRemoved: string;
```

Comment:
minor: why are you declaring these independently? couldn't you just do the following?

```
const [ templateMetadata ] = extract...
```

Reply:
I seem to recall the compiler complaining at me about that. 🤔 
But more likely, my limited TypeScript knowledge could have messed me up and I could have been doing something else wrong that caused the compiler to yell at me.

---

At any rate, it works as you suggested so I've made the change. Thanks. 👍